modules/wpml/wpml-config.php

```diff
@@ -262,6 +286,101 @@ public function translate_taxonomies( $taxonomies, $hide ) {
 		return $taxonomies;
 	}
 
+	/**
+	 * Translation management for strings in blocks content.
+	 *
+	 * @since 3.3
+	 *
+	 * @param string[][] $parsing_rules Rules as Xpath expressions to evaluate in the blocks content.
+	 * @return string[][] Rules completed with ones from wpml-config file.
+	 *
+	 * @phpstan-param array<string,array<string>> $parsing_rules
+	 * @phpstan-return array<string,array<string>>
+	 */
+	public function translate_blocks( $parsing_rules ) {
+		return array_merge( $parsing_rules, $this->get_blocks_parsing_rules( 'xpath' ) );
+	}
+
+	/**
+	 * Translation management for blocks attributes.
+	 *
+	 * @since 3.3
+	 *
+	 * @param string[][] $parsing_rules Rules for blocks attributes to translate.
+	 * @return string[][] Rules completed with ones from wpml-config file.
+	 *
+	 * @phpstan-param array<string,array<string>> $parsing_rules
+	 * @phpstan-return array<string,array<string>>
+	 */
+	public function translate_blocks_attributes( $parsing_rules ) {
+		return array_merge( $parsing_rules, $this->get_blocks_parsing_rules( 'key' ) );
+	}
+
+	/**
+	 * Returns rules to extract a kind of string for blocks.
+	 *
+	 * @since 3.3
+	 *
+	 * @param string $rule_tag Tag name to extract.
+	 * @return string[][] Rules completed with ones from wpml-config file.
```

Comment:
```suggestion
	 * Returns rules to extract translatable strings from blocks.
	 *
	 * @since 3.3
	 *
	 * @param string $rule_tag Tag name to extract.
	 * @return string[][] The rules.
```